internal/bminventory/inventory.go

```diff
@@ -2787,6 +2788,20 @@ func (b *bareMetalInventory) getOLMOperators(cluster *common.Cluster, newOperato
 		if err != nil {
 			return nil, common.NewApiError(http.StatusBadRequest, err)
 		}
+
+		// TODO - Need to find a better way for creating LVMO/LVMS operator on different openshift-version
+		if operator.Name == "lvm" {
+			lvmsMetMinOpenshiftVersion, err := common.VersionGreaterOrEqual(cluster.OpenshiftVersion, lvm.LvmsMinOpenshiftVersion)
+			if err != nil {
+				return nil, err
```

Comment:
not sure that we want to return nil - maybe we should set lvms as the default and write error to log.
Another request - please make sure that we are not calling this method in day 2 because we don't have OCP version and it can be nil